### PR TITLE
envoy 1.19.1

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Created by https://github.com/mislav/bump-homebrew-formula-action

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
@chenrui333 ps I switched the syntax to favor automation

File: Formula/envoy.rb [hunk 1]
```diff
@@ -42,7 +41,7 @@ class Envoy < Formula
   # Work around xcode 12 incompatibility until envoyproxy/envoy#17393
   patch do
     url "https://github.com/envoyproxy/envoy/commit/3b49166dc0841b045799e2c37bdf1ca9de98d5b1.patch?full_index=1"
-    sha256 "e65fe24a29795606ea40aaa675c68751687e72911b737201e9714613b62b0f02"
+    sha256 "9a7933245b40dc1b72981b0581f9a0a37c3d450ea9e804823a43ecdee9827cd8"
```
Comment (codefromthecrypt) [inline Formula/envoy.rb:44]:
```suggestion
    sha256 "e65fe24a29795606ea40aaa675c68751687e72911b737201e9714613b62b0f02"
```

**Comment by codefromthecrypt** [comment]
Also I looked at v1.19.1 doesn't include the patch even if the PR was merged. I don't know why but envoy usually doesn't cherry-pick so probably v1.20.0 will be the first version we can kill the patch with

**Comment by codefromthecrypt** [comment]
what an odd failure.. I wonder if it is transient (on ubuntu)

```
[0 / 4] [Prepa] BazelWorkspaceStatusAction stable-status.txt ... (2 actions, 0 running)
ERROR: <builtin>: BazelWorkspaceStatusAction stable-status.txt failed: Failed to determine workspace status: Process exited with status 1
fatal: not a git repository (or any of the parent directories): .git
Target //source/exe:envoy-static failed to build
INFO: Elapsed time: 61.624s, Critical Path: 0.07s
```

**Comment by codefromthecrypt** [comment]
force pushed just in case the problem is transient

**Comment by codefromthecrypt** [comment]
no luck on this being transient. Can I call in an escalation? maybe @keith to help try to get 1.19.1 building? seems 1.18.4 worked alright on linux, too. #83929

**Comment by cho-m** [comment]
The error seems to that `envoy` build is checking for Git information; however, GitHub generated source tarballs usually exclude this.

I believe the following script is run via Bazel's `--workspace_status_command`: https://github.com/envoyproxy/envoy/blob/main/bazel/get_workspace_status

The easiest way to fix would be to switch back to direct Git clone rather than source tarball.

**Comment by codefromthecrypt** [comment]
thanks for the analysis @cho-m so we can switch back I will make a comment basically we can't use automatic update scripts until this is fixed I think.

**Review by carlocab — APPROVED** [review]
We can nix the comment in a separate syntax-only PR, as this CI run is pretty expensive.

**Pull request closed without merging**